client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -108,6 +109,13 @@
      */
     private final MetricListener metricListener;
 
+    /**
+     * If the flag is set, RawClient will timeout the future if it doesn't receive response from Segment store
+     * in a given time period. Notice that we don't want this feature enabled in testing since it might mask some
+     * bugs.
+     */
+    private final Duration rawclientTimeout;
```

Comment:
This is a public api, so it needs to make sense to a user. RawClient is not a public class, so this explication makes no sense. It would be better if this was "serverRequestTimeout" or something similar.

Reply:
Got it! Makes sense.